File: Day 08/homework/hw7.py
# შეამოწმე, არის თუ არა ციფრი მთელი რიცხვი. 

num = input("enter a  number: ")
num1 = (int(num)) or (float(num))
if num1 == int(num):
    print("whole number")
else:
    print("error")

File: Day 64/homework/hw7.js
/*შემოიტანე 2 input შეამოწმე, არის თუ არა ორი რიცხვის ჯამი 100-ზე მეტი.*/

var num1 = prompt("enter number")
var num2 = prompt("enter number")

num1 = Number(num1);
num2 = Number(num2);

if (num1 + num2 > 100){
    console.log("sum > 100")
} else{
    console.log("sum < 100")
}